releasenotes/notes/fix-integer-quadratic-lp-bug-ad4faf5379ef7bba.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+fixes:
+  - |
+    Fix misspelled `upper_bound` in `from_lp_file` constructor method when
```

Comment:
I think something like
```
No longer throw a ``TypeError`` from ``ConstrainedQuadraticModel.from_lp_file`` when adding a integer quadratic term from constraints.
```
would be clearer, or at least describes the broken behaviour